src/main/java/zd/zero/waifu/motivator/plugin/actions/MotivateMeAction.java

```diff
@@ -22,8 +24,11 @@ public void actionPerformed( @NotNull AnActionEvent e ) {
             pluginState.isMotivateMeEnabled(),
             pluginState.isMotivateMeSoundEnabled() );
 
-        WaifuMotivation waifuMotivation = TextualMotivationFactory.getInstance().constructMotivation( e.getProject(),
```

Comment:
I really did not want to maintain the code that handled the bundled motivation assets, so I just replaced it with temporary fix. The motivate me feature will be enhanced when #157 gets worked.

Also `TextualMotivationFactory` and it's dependencies are now un-used, should I remove?

Reply:
> Also TextualMotivationFactory and it's dependencies are now un-used, should I remove?

Yes please, we can retrieve the class from VCS in case we need it.

Reply:
Dun!